### ISSUE TITLE
How to set DecisionTreeClassifier reproducibility?

### ISSUE THREAD

**Issue by alvgoro** [javascriptdata/scikit.js#206]
Hi, first of all, thanks for developing this library :)

I was writing a code that use the `DecisionTreeClassifier`. When I checked my results, I realised that they are not constants.
I mean, I set a random number generator with `Math.seedrandom(my_seed)` to get the same results for each execution, but they are not.

This method worked for me on `LogisticRegression`, where the weights are initialized by TensorflowJS (if I am not wrong, the user can "fixs" this initialization using `Math.seedrandom()` before building the neural network). I know DecisionTreeClassifier is not a neural network, so the random number generation is not equal to LogisticRegression.

In summary: How can I set a random number generator for getting the same results on a DecisionTreeClassifier?
